src/cca/app/scripts/cordova_browser_api.ts

```diff
@@ -343,6 +347,14 @@ class CordovaBrowserApi implements BrowserAPI {
     }
   }
 
+  public exit = () => {
+    (<any>navigator).Backbutton.goBack(() => { /* MT (success) */ }, () => {
```

Comment:
Definitely a good idea.  It looked like other files were per-navigator-property so I did the same, can you confirm that this looks good?